### DISCUSSION TITLE
Official / Recommended Docker image to use ?

### DISCUSSION THREAD

**Discussion by nodecentral** [tfeldmann/organize#273] [Q&A]
Hi

I can see a couple of docker instances online, is there one that is recommended ?

https://github.com/dskaggs/docker-organize
https://github.com/wintuxx/docker-organize

**Comment by tfeldmann** [comment]
No official recommendation - the one from wintuxx is a fork of dskaggs' image with some modifications. Another docker file lives in PR #65.

**Comment by nodecentral** [comment]
Thanks @tfeldmann 

Both links don’t seem to have any specific install instructions, not sure if you can help ? I did the following.
````
cd tmp
````
````
git clone https://github.com/wintuxx/docker-organize.git
````
````
cd docker-organize
````
````
sudo docker build -t docker-organize .
````
Which created the following
````
[/tmp/docker-organize] # ls
docker-compose.yml  Dockerfile  LICENSE  README.md  sample_config.yml  scripts/
````
Which created the following
````
[/tmp/docker-organize] # ls
docker-compose.yml  Dockerfile  LICENSE  README.md  sample_config.yml  scripts/
````

Then I tried to run it,  but no luck..
````
docker run -it \
	-v "/share/Container/organize/config/":/home/pi/.config/organize/ \
	-v "/share/Container/organize/logs/":/var/log/organize/ \
	-v "/share/Container/organize/input/":/Input \
	-v "/share/Container/organize/output/":/Output \
	docker-organize
````
